doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -233,8 +233,13 @@ The configured cache memory budget can be used in a few different ways:
 Cache memory usage is governed by the following options:
 ``bluestore_cache_meta_ratio``, ``bluestore_cache_kv_ratio``, and
 ``bluestore_cache_kv_max``.  The fraction of the cache devoted to data
-is 1.0 minus the meta and kv ratios.  The memory devoted to kv
-metadata (the RocksDB cache) is capped by ``bluestore_cache_kv_max``
+is governed by the effective bluestore cache size (depending on
+``bluestore_cache_size[_ssd|_hdd]`` settings and the device class of the primary
+device), the meta and kv ratios and the ``bluestore_cache_kv_max`` setting.
+The data fraction can be calculated by
+``<effective_cache_size> - bluestore_cache_meta_ratio * <effective_cache_size> -
+min(bluestore_cache_kv_ratio * <effective_cache_size>, bluestore_cache_kv_max)``
```

Comment:
bluestore_cache_kv_max isn't used any more and formula is a bit simpler now:
<effective_cache_size> - bluestore_cache_meta_ratio * <effective_cache_size> -
bluestore_cache_kv_ratio * <effective_cache_size>

@markhpc - haven't I missed anything?

Reply:
@ifed01 updated.